client/src/pages/book/BookRanking.tsx

```diff
@@ -135,12 +178,18 @@ export default function BookRanking() {
               <FormControl>
                 <InputLabel id="ranking-direction-label">Sortuj</InputLabel>
                 <Select
-                  value={sortDirection}
+                  fullWidth
+                  value={sortValue}
                   label="Sortuj"
-                  onChange={handleSortDirectionChange}
+                  onChange={(e: SelectChangeEvent<string>) => {
+                    setSortValue(e.target.value);
```

Comment:
Nie wiem czemu ten setSortValue nie działa, to znaczy w tym momencie przy zmianie sortowania zmienia się ranking, ale label selectu od sortowania już nie